integration-tests/tests.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+
+import subprocess, argparse, shlex, sys, re, os
+from time import sleep
+
+run_options = ['put','get','exist']
+
+def run_tests(behaviour, args):
+    try:
+        print(f'Running bobp -b {str(behaviour)} {args.rstrip()}')
+        p = subprocess.check_output(shlex.split(f'./bobp -b {behaviour} {args.rstrip()}')).decode('ascii')
+        print(str(p))
+        if behaviour in {'put', 'get'}:
+            if f'{behaviour} errors:' in str(p) or f'panicked' in str(p):
```

Comment:
Same here:
It is better to check for success, not for errors. Because if the output of `bobp` someday will change, the test start passing by even on errors. I suggest to search for `total err: 0`